.drone.yml

```diff
@@ -61,6 +61,16 @@ pipeline:
     when:
       event: [ push, tag, deployment ]
 
+  test_java11:
+    group: test
+    image: lampepfl/dotty:2019-04-22
+    commands:
```

Comment:
We should still set the `PATH` since some of our tests will invoke `javac` for example.